experiments/benchmarks/bucket.jl

```diff
@@ -0,0 +1,214 @@
+# # Global bucket run using temporal map albedo
```

Comment:
This looks good. Do we want to remove the profile/flame graph stuff here then? https://github.com/CliMA/ClimaLand.jl/blob/main/experiments/standalone/Bucket/global_bucket_temporalmap.jl

I am guessing we may still want to keep this running in experiments/standalone/Bucket, since it also uploads plots there, but we could also consider removing it entirely from there?


More broadly, should any script that we do the profile/allocations on should be moved here?

Reply:
I am thinking that this benchmark job should be the ground truth of performance/allocation, but it doesn't have to be the only job that provides information about performance. We can remove the steps from the other bucket jobs, but I think it is useful to have them run all the time so that we can track regressions if they occur.

Reply:
ok! we can leave as is then. Thank you!